### PR TITLE
Improve EVM functionality

### PR DESCRIPTION
**Pull request by jayantk**
I'm trying to get this to a point where we can document the EVM contract. This PR does a couple small things:
1. adds an EVM network selector
2. fixes the decoding of evm results and renders them properly
3. cleans up how dynamic values are rendered for code / examples.
4. lays out the file structure for the evm docs

### CODE REVIEW

File: abis/PythErrors.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
+[
```
Comment (jayantk) [inline abis/PythErrors.json:1]: you need the error abi to decode the names of evm errors

**Review by cctdaniel — APPROVED** [review]
I'll add vercel preview deployment in a separate PR to make reviewing PRs easier

**Comment by cctdaniel** [comment]
seems like there are some build errors due to types

**Comment by jayantk** [comment]
yeah, i fixed the type errors in the docker branch, so merging handled it here.

**Pull request merged**